package.json

```diff
@@ -73,7 +73,7 @@
     "ts-jest": "^27.1.3",
     "ts-json-schema-generator": "^0.97.0",
     "ts-node": "^10.4.0",
-    "typescript": "4.6.4",
+    "typescript": "4.9.4",
```

Comment:
Please do not update typescript versions. We fixate these on purpose

Reply:
This was reverted. Please resolve.